react-frontend/src/css/nav.css

```diff
@@ -17,7 +17,23 @@ header {
     margin: auto;
     width: 1115px;
   }
-  
+
+  .covidBanner {
+    background-color: #fcba19;
+    top: 0;
+    position: fixed;
+    height: 6%;
+    width: 100%;
+    padding: 0 0 0 200px;
```

Comment:
Don't think we should attempt to achieve layout this way. This will cause the close button to be positioned really weirdly on smaller displays, for example:

![Screen Shot 2020-09-21 at 4 13 02 PM](https://user-images.githubusercontent.com/16946297/93830608-81ce6c80-fc25-11ea-9ead-d81d87f7419c.png)


Would be better I think, as on gov.bc.ca, to put a `<div class="container">` inside of `.covidBanner` to enforce a maximum width.